### PR TITLE
Chore/i18n optimization

### PR DESCRIPTION
**Pull request by thiagomurakami**
#### What is the purpose of this pull request?
Use `react-intl`'s `defineMessages` and `FormattedMessage` to optimize messages.

#### What problem is this solving?
This should improve performance according to @tlgimenes 

#### Types of changes
- [x] VTEX IO necessary changes.

PS: There are two messages that still remain:
```
	admin/pages.admin-menu-button.pages
	admin/pages.admin-menu-button.storefront
```

### CODE REVIEW

**Comment by thiagomurakami** [comment]
I renamed some files to `.tsx` because the `defineMessages` in `.ts` files weren't being parsed, it's probably a bug in the babel plugin that is used.

File: react/components/EditorContainer/StoreEditor/Styles/StyleEditor/mutations/RenameStyle.tsx [hunk 1]
```diff
@@ -1,4 +1,3 @@
-import React from 'react'
```
Comment (alcar) [inline react/components/EditorContainer/StoreEditor/Styles/StyleEditor/mutations/RenameStyle.tsx]: If you don't need React (nor JSX), then it should be a `.ts` file.

File: react/components/EditorContainer/StoreEditor/Styles/StyleEditor/mutations/UpdateStyle.tsx [hunk 2]
```diff
@@ -1,4 +1,3 @@
-import React from 'react'
```
Comment (alcar) [inline react/components/EditorContainer/StoreEditor/Styles/StyleEditor/mutations/UpdateStyle.tsx]: If you don't need React (nor JSX), then it should be a `.ts` file.

File: react/components/EditorContainer/StoreEditor/Styles/StyleEditor/queries/GenerateStyleSheet.tsx [hunk 3]
```diff
@@ -1,4 +1,3 @@
-import React from 'react'
```
Comment (alcar) [inline react/components/EditorContainer/StoreEditor/Styles/StyleEditor/queries/GenerateStyleSheet.tsx]: If you don't need React (nor JSX), then it should be a `.ts` file.

File: react/components/EditorContainer/StoreEditor/Styles/StyleList/mutations/CreateStyle.tsx [hunk 4]
```diff
@@ -1,4 +1,3 @@
-import React from 'react'
```
Comment (alcar) [inline react/components/EditorContainer/StoreEditor/Styles/StyleList/mutations/CreateStyle.tsx]: If you don't need React (nor JSX), then it should be a `.ts` file.

File: react/components/EditorContainer/StoreEditor/Styles/StyleList/mutations/DeleteStyle.tsx [hunk 5]
```diff
@@ -1,4 +1,3 @@
-import React from 'react'
```
Comment (alcar) [inline react/components/EditorContainer/StoreEditor/Styles/StyleList/mutations/DeleteStyle.tsx]: If you don't need React (nor JSX), then it should be a `.ts` file.

File: react/components/EditorContainer/StoreEditor/Styles/StyleList/mutations/SaveSelectedStyle.tsx [hunk 6]
```diff
@@ -1,4 +1,3 @@
-import React from 'react'
```
Comment (alcar) [inline react/components/EditorContainer/StoreEditor/Styles/StyleList/mutations/SaveSelectedStyle.tsx]: If you don't need React (nor JSX), then it should be a `.ts` file.

File: react/components/EditorContainer/StoreEditor/Styles/StyleList/queries/ListStyles.tsx [hunk 7]
```diff
@@ -1,4 +1,3 @@
-import React from 'react'
```
Comment (alcar) [inline react/components/EditorContainer/StoreEditor/Styles/StyleList/queries/ListStyles.tsx]: If you don't need React (nor JSX), then it should be a `.ts` file.

**Pull request merged**